jaeger-core/src/main/java/io/jaegertracing/internal/JaegerObjectFactory.java

```diff
@@ -62,9 +62,21 @@ public JaegerSpanContext createSpanContext(
       long spanId,
       long parentId,
       byte flags,
+      String traceState,
       Map<String, String> baggage,
       String debugId) {
-    return new JaegerSpanContext(traceIdHigh, traceIdLow, spanId, parentId, flags, baggage, debugId, this);
+    return new JaegerSpanContext(traceIdHigh, traceIdLow, spanId, parentId, flags, traceState, baggage, debugId, this);
+  }
+
+  public JaegerSpanContext createSpanContext(
```

Comment:
Please keep the factory interface minimal.

Can we instead have a setTracestate() method on the context and not change the factory? I realize that we can't make the field final in that case, but it's better for backwards compatibility than breaking this interface.

Reply:
`JaegerObjectFactory` is not an interface hence this is not a breaking change.